README.md

```diff
@@ -53,7 +54,7 @@ Note that tgstation-server has only ever been tested on Linux via it's [docker e
 
 #### Docker (Linux)
 
-tgstation-server supports running in a docker container and is the recommended deployment method for Linux systems. The official image repository is located at https://hub.docker.com/r/tgstation/server. It can also be built locally by running `docker build . -f build/Dockerfile -t <your tag name>` in the repository root.
+tgstation-server supports running in a docker container although it is only recommended if you are having issues running it natively. The official image repository is located at https://hub.docker.com/r/tgstation/server. It can also be built locally by running `docker build . -f build/Dockerfile -t <your tag name>` in the repository root.
```

Comment:
I'd still argue that it is the recommended method for running TGS, to be honest.